source/test/F0.Minesweeper.Components.Tests/Pages/Game/Modules/CellTests.razor

```diff
@@ -157,54 +142,23 @@
-	[InlineData(CellInteractionType.LeftClick, false, CellStatusType.Uncovered, 3, '3', "f0-cell f0-cell-uncovered")]
-	[InlineData(CellInteractionType.LeftClick, false, CellStatusType.Uncovered, 4, '4', "f0-cell f0-cell-uncovered")]
-	[InlineData(CellInteractionType.LeftClick, false, CellStatusType.Uncovered, 5, '5', "f0-cell f0-cell-uncovered")]
-	[InlineData(CellInteractionType.LeftClick, false, CellStatusType.Uncovered, 6, '6', "f0-cell f0-cell-uncovered")]
-	[InlineData(CellInteractionType.LeftClick, false, CellStatusType.Uncovered, 7, '7', "f0-cell f0-cell-uncovered")]
-	[InlineData(CellInteractionType.LeftClick, false, CellStatusType.Uncovered, 8, '8', "f0-cell f0-cell-uncovered")]
-	[InlineData(CellInteractionType.Automatic, false, CellStatusType.Uncovered, 0, '0', "f0-cell f0-cell-uncovered")]
-	[InlineData(CellInteractionType.Automatic, false, CellStatusType.Uncovered, 1, '1', "f0-cell f0-cell-uncovered")]
-	[InlineData(CellInteractionType.Automatic, false, CellStatusType.Uncovered, 2, '2', "f0-cell f0-cell-uncovered")]
-	[InlineData(CellInteractionType.Automatic, false, CellStatusType.Uncovered, 3, '3', "f0-cell f0-cell-uncovered")]
-	[InlineData(CellInteractionType.Automatic, false, CellStatusType.Uncovered, 4, '4', "f0-cell f0-cell-uncovered")]
-	[InlineData(CellInteractionType.Automatic, false, CellStatusType.Uncovered, 5, '5', "f0-cell f0-cell-uncovered")]
-	[InlineData(CellInteractionType.Automatic, false, CellStatusType.Uncovered, 6, '6', "f0-cell f0-cell-uncovered")]
-	[InlineData(CellInteractionType.Automatic, false, CellStatusType.Uncovered, 7, '7', "f0-cell f0-cell-uncovered")]
-	[InlineData(CellInteractionType.Automatic, false, CellStatusType.Uncovered, 8, '8', "f0-cell f0-cell-uncovered")]
-	[InlineData(CellInteractionType.GameLost, true, CellStatusType.Mine, 8, '☢', "f0-cell f0-cell-uncovered")]
-	public void SetUncoveredStatus_StatusManagerCanMoveNext_ChangesTextToCorrectTranslation(CellInteractionType cellInteraction, bool isMine, CellStatusType newStatus, byte adjacentMineCount, char expectedStatusChar, string expectedCssClass)
-	{
-		// Arrange
-		RenderFragment expectedMarkup = @<div><button class="@expectedCssClass">@expectedStatusChar</button></div>;
-			
-		cellStatusManagerMock.Setup((manager) => manager.CanMoveNext(cellInteraction, isMine)).Returns(true);
-		cellStatusManagerMock.Setup((manager) => manager.MoveNext(cellInteraction, isMine)).Returns(newStatus);
-
-		ComponentParameter parameter = ComponentParameterFactory.Parameter(nameof(Cell.Location), new Location(1, 1));
+        CellInteractionType cellInteraction = CellInteractionType.LeftClick;
+        bool isMine = true;
+        CellStatusType newStatus = CellStatusType.MineExploded;
+        byte adjacentMineCount = 3;
+        char expectedStatusChar = '☢';
+        string expectedCssClass = "f0-cell f0-cell-mine-exploded";
+
+        RenderFragment expectedMarkup = @<div><button class="@expectedCssClass">@expectedStatusChar</button></div>
+    ;
```

Comment:
Typographically, this is either a [widow or orphan](https://de.wikipedia.org/wiki/Hurenkind_und_Schusterjunge), not sure. I linked the German _Wikipedia_ article due to the "explicitness" of these terms 😄.

Reply:
I don't know why that's formatted in that way by the IDE so I sadly don't know how to fix that permanently on my end. So for now I put it manually back where it belongs.